spec/unit/pops/types/enumeration_spec.rb

```diff
@@ -1,50 +1,51 @@
 require 'spec_helper'
 require 'puppet/pops'
 
+module Puppet::Pops::Types
```

Comment:
This makes me nervous - I don't trust rspec/mocha well enough to be confident it does not leave stuff behind in the module that later may affect other tests.
I am probably being paranoid. Ping @kylog have we used this construct elsewhere?
@thallgren do we really have to do this?